nextcord/channel.py

```diff
@@ -1411,6 +1411,53 @@ def archived_threads(
             self.id, self.guild, limit=limit, joined=joined, private=private, before=before
         )
 
+    async def create_webhook(
+        self,
+        *,
+        name: str,
+        avatar: Optional[Union[bytes, Asset, Attachment, File]] = None,
+        reason: Optional[str] = None,
+    ) -> Webhook:
+        """|coro|
+        Creates a webhook for this channel.
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
+
+        ..versionadded:: 2.4
```

Comment:
```suggestion
        .. versionadded:: 2.4
```
![image](https://user-images.githubusercontent.com/34910574/205679459-f8af0ed4-8233-4b85-abe3-3e59ae75a70e.png)
Spacing matters.